README.md

```diff
@@ -2,12 +2,93 @@
 
 This repo contains all the information needed for both in-person and remote participation in the event.
 
+## Before the Event
+
+TODO 
+
+## During the Event
+
+1. **Teams**: 
+    1. Make sure to have a team. 
+    2. We want to bring together: developers, PMs, Data scientists, designers etc. 
+    3. A team could be made up of engineers only or a mix of cross-discipline teammates. 
+    4. We encourage freedom to innovate and be creative. 
+    5. Team Size Recommendation: 3 to 5 people.
+2. **Logistics:**
+    6. Slack will be the main communication channel for the event. Please use [#hackathon-ga](https://codeforvenezuela.slack.com/messages/CH5JMFBB2) to post questions and requests. 
+    7. When there are talks or workshops, we will make sure we have a video stream available for remote participants.
+
 ## Challenges
-In the `challenges` folder you will find a list of challenges proposed by our partner organizations. Each challenge will have a unique identifier code and will have it's own channel in Slack where you will be able to ask questions.
- 
+
+The focus of the event would be in the following areas:
+
+* Using data to analyze, understand and communicate the Venezuelan crisis.
+* Digital products and platforms to connect the diaspora with the country.
+
+1. In the `challenges` folder you will find a list of challenges proposed by our partner organizations. Please take time to review them before the event. The more prepared, the more chances your team will have of making good progress during the weekend! 
+2. Each challenge will have a dedicated Slack channel where you could ask questions and get clarifications. The slack channel for a channel follows the following convention: chal-*challenge_code*. (e.g #[chal-mpv-inf](https://codeforvenezuela.slack.com/messages/CHEG10M7S), [#chal-mit-rea](https://codeforvenezuela.slack.com/messages/CH875RP7T))
+
+If you want to work on a challenge that is not by our partner organization, you are welcome to do so! Share you challenge in [#hackathon-ga](https://codeforvenezuela.slack.com/messages/CH5JMFBB2).
+
+## Demos and Presentations
+
+1. Each team will present a 3 minute demo on their work. 
+2. You can record a video and we will present it on site as a part of the final presentations. 
+3. **Deadline**: Sunday 14, 4:30pm Pacific Time) 
+4. **Software**:
+    1. Our suggestion is to use Loom to record the video of the demo: [https://www.useloom.com](https://www.useloom.com)
+5. Post your video via Slack in the channel [#hackathon-ga](https://codeforvenezuela.slack.com/messages/CH5JMFBB2) 
+6. If possible, you are invited to watch live the feedback from our panelist. This will happen around 4:30PM PST,  Sunday 14.
```

Comment:
I would remove the `If possible` here. We are inviting folks to watch the live feedback, they can decide whether they would take that invitation.